docs/src/pages/docs/comparison.md

```diff
@@ -8,44 +8,53 @@ title: Comparison
+|                                              | React Query                            | SWR [_(Website)_][swr]     | Apollo Client [_(Website)_][apollo]   | Relay [_(Website)_][apollo]           |
+| -------------------------------------------- | -------------------------------------- | -------------------------- | ------------------------------------- | ------------------------------------- |
+| Supported Query Syntax                       | Promise, REST, GraphQL                 | Promise, REST, GraphQL     | GraphQL                               | GraphQL                               |
+| Supported Query Keys                         | JSON                                   | JSON                       | GraphQL Query                         | GraphQL Query                         |
+| Query Key Change Detection                   | Deep Compare (Stable Serialization)    | Referential Equality (===) | Deep Compare (Unstable Serialization) | Deep Compare (Unstable Serialization) |
+| Query Data Memoization Level                 | Query + Structural Sharing             | Query                      | Query + Entity + Structural Sharing   | Query + Entity + Structural Sharing   |
+| Bundle Size                                  | [![][bp-react-query]][bpl-react-query] | [![][bp-swr]][bpl-swr]     | [![][bp-apollo]][bpl-apollo]          | [![][bp-react-relay]][bpl-react-relay]|
+| Queries                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Caching                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Devtools                                     | ✅                                     | 🟡                         | ✅                                    | ✅                                    |
+| Polling/Intervals                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Parallel Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Dependent Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Paginated Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Infinite Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
```

Comment:
Also in the new beta comparison is "Infinite Query Refetching". To explain better, if you have added more data to an infinite list, let's say "next, next, previous, next, previous" or any order/permutation, the ability to refresh the data would need to take into account that exact order and use/update the cursors and/or variables used to fetch each action, then ultimately "replay" those actions to refresh the final window of data.

I have not yet heard of or seen another library that does this out of the box, and while I wouldn't be surprised if Relay supported it, I am doubtful it does currently. Let's verify this one as well.

Reply:
Interesting! I'm not sure, I think thats where GraphQL connections are for, but I only see refetchConnection in Relay as whole not partly for 1 page.

Reply:
Hmmm. Honestly, I think we could definitely use a ❔ icon in the key for things that still need to be verified. I don't want any of these icons to be guessing TBH.